### DISCUSSION TITLE
`isItemDisabled` prop for Cards component ?

### DISCUSSION THREAD

**Discussion by naveen8801** [cloudscape-design/components#2053] [Q&A] [answered]
How can I use the `isItemDisabled` prop for Cards component ? , is there any demo around this or any documentation ?

**Comment by taheramr** [comment, marked as answer]
You can find more about this property here: https://cloudscape.design/components/cards/?tabId=api#:~:text=Determines%20which%20items%20are%20disabled.%20If%20an%20item%20is%20disabled%2C%20users%20can%27t%20select%20it.

It basically runs against the `items` passed to the Cards component and based on what it returns it either disable the card with that specific item or not, its more like the `array.filter` function that receives a callback.

Let us know if this is helpful!

**Reply by Al-Dani** [reply]
We don't have it in our demos, but you can play around with it on our [website](https://cloudscape.design/components/cards/?tabId=playground). 

Small example: disable selection for the cards of a certain type
`isItemDisabled={item => item.type === "2A"}`
